js/about/preferences.js

```diff
@@ -722,7 +722,17 @@ class TabsTab extends ImmutableComponent {
 class PaymentsTab extends ImmutableComponent {
   constructor () {
     super()
+    this.state = {
```

Comment:
please put the state in `AboutPreferences` and pass it to `PaymentTab` as props.